.dockerignore

```diff
@@ -2,3 +2,5 @@
 !assembly/assembly-main/target/eclipse-che-*
 assembly/assembly-main/target/eclipse-che-*.tar.gz
 assembly/assembly-main/target/eclipse-che-*.zip
+
+!assembly/assembly-main/src/assembly/bin/launcher.sh
```

Comment:
The first line of the .dockerignore is `*` (exclude all files). So we don't need to add new lines for new directories.